Enhance Prometheus metrics section for Redis Cloud (#2173)
redis/docs: content/integrate/prometheus-with-redis-cloud/_index.md

The Redis Cloud Prometheus endpoint is exposed on Redis Cloud's internal network. To access this network, enable [VPC peering]({{< relref "/operate/rc/security/vpc-peering" >}}) or [Private Service Connect]({{< relref "/operate/rc/security/private-service-connect" >}}). Both options are only available with Redis Cloud Pro. You cannot use Prometheus and Grafana with Redis Cloud Essentials.

  {{< note >}}
  

>   The Prometheus endpoint is scoped per subscription. Metrics available on this endpoint cover all databases within that subscription. Any database added to or removed from the subscription is reflected automatically on the endpoint. 

Reviewer: ```suggestion
The Prometheus endpoint's metrics cover all databases within a subscription. Any database added to or removed from the subscription is reflected automatically. 
```
